ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/SvarUtService.java

```diff
@@ -0,0 +1,23 @@
+package no.difi.meldingsutveksling.ks;
+
+import no.difi.meldingsutveksling.core.EDUCore;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SvarUtService {
+    public EDUCoreConverter messageConverter;
+    public SvarUtWebServiceClient client;
+
+    public SvarUtService(EDUCoreConverter messageConverter, SvarUtWebServiceClient client) {
+        this.messageConverter = messageConverter;
+        this.client = client;
+    }
+
+    public String send(EDUCore message) {
+        final Forsendelse forsendelse = messageConverter.convert(message);
+
+        final String forsendelseId = client.sendMessage(forsendelse);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "forsendelseId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1488)